### PR TITLE
Add ton prices

### PR DESCRIPTION
**Pull request by kooroot**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1625,7 +1625,8 @@ FROM
     ('val-valeria', 'ethereum', 'VAL', 0x011e128ec62840186f4a07e85e3ace28858c5606, 18),
     ('propc-propchain', 'ethereum', 'PROPC', 0x9ff58067bd8d239000010c154c6983a325df138e, 18),
     ('beam-beam-eth', 'ethereum', 'BEAM', 0x62d0a8458ed7719fdaf978fe5929c6d342b0bfce, 18),
-    ('sdl-stakelink', 'ethereum', 'SDL', 0xa95c5ebb86e0de73b4fb8c47a45b792cfea28c23, 18)
+    ('sdl-stakelink', 'ethereum', 'SDL', 0xa95c5ebb86e0de73b4fb8c47a45b792cfea28c23, 18),
+    ('ton-tokamak-network', 'ethereum', 'TON', 0x2be5e8c109e2197D077D13A82dAead6a9b3433C5, 18)
```
Comment (Hosuke) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1629]: API: https://api.coinpaprika.com/v1/coins/ton-tokamak-network

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @kooroot 

In this case, we may close PR #4732 .

File: models/prices/ethereum/prices_ethereum_tokens.sql (hunk 1 above)
Comment (kooroot) [reply models/prices/ethereum/prices_ethereum_tokens.sql:1629]: Thanks~ @Hosuke

Comment (Hosuke) [reply models/prices/ethereum/prices_ethereum_tokens.sql:1629]: Sorry @kooroot, this PR is not merged yet. You may need to reopen this PR.

**Pull request closed without merging**